services/sinner/graphql/server/graph/helper.go

```diff
@@ -0,0 +1,53 @@
+package graph
+
+import (
+	"github.com/synapsecns/sanguine/services/sinner/db/model"
+	graphqlmodel "github.com/synapsecns/sanguine/services/sinner/graphql/server/graph/model"
+)
+
+// Helper function to convert int to *int.
+func intPtr(val int) *int {
+	return &val
+}
+
+// Helper function to convert string to *string.
+func strPtr(val string) *string {
+	return &val
+}
+
+func dbToGraphqlModelOrigin(event model.OriginSent) *graphqlmodel.OriginInfo {
+	return &graphqlmodel.OriginInfo{
+		MessageHash:        &event.MessageHash,
```

Comment:
How would you implement? with namer?

Reply:
nah, I'm saying because you pass by reference rather than `core.PtrTo`, (or something that clones rather than borrows the string) , mutations to the original object (event) will modify the underlying object.

I wouldn't expect any here except in helper.go you *also* use a pointer for a different string also from here and I'm not sure why